spec/student_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../person'
+require_relative '../student'
+
+describe Student do
```

Comment:
- [ Required ] I think it would be better if you add tests for the `play_hooky` and `classroom` methods in the `Student` class, according to the project requirements you are expected to test all methods in the classes you created including the `Classroom` class. Please see the screenshot below.

## Project requirements
![Screenshot from 2023-06-09 00-19-48](https://github.com/LoviMadaan/oop-school-library/assets/16392046/e50b3362-7540-4d9b-a450-83a1b4c934ba)